apps/site-app/src/translations/en.json

```diff
@@ -41,7 +41,6 @@
   "'InBntU'": "Settings",
   "'It9lF6'": "Settings",
   "'JHN0Qb'": "Transfer funds from your Canadian bank to your account.",
-  "'JgdFdU'": "Profile",
   "'K7rhf2'": "Settings",
```

Comment:
We should not commit those they should be compiled when build. It could become a conflict problem and it pollutes git.

Reply:
yeah, it's a real pain.  Also guaranteed conflicts when anything changes.  As I'm going through the translation PR's I'm removing these.